api-catalog-ui/frontend/cypress/support/commands.js

```diff
@@ -25,3 +25,16 @@
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
 import 'cypress-file-upload';
+
```

Comment:
thanks for generalizing this command. I think we can also use it in the remaining tests `login-ok.cy.js` and `login-bad.cy.js`

Reply:
Done. Custom 'login' command used in the `login-ok.cy.js` and `login-bad.cy.js`